### ISSUE TITLE
[SUPPORT]Fine-tuning notebook didn't return  validation loss

### ISSUE THREAD

**Issue by Lauorie** [openai/openai-cookbook#659]
https://github.com/openai/openai-cookbook/blob/main/examples/How_to_finetune_chat_models.ipynb

This notebook is great, but how can we return validation loss to see if the model overfits the training set?

And in "Once it is completed, you can use the **result_files** to sample the results from the validation set (if you uploaded one), and use the ID from the fine_tuned_model parameter to invoke your trained model.", what's in the **result_files**? How can I return it? THX
